benchmarks/replay/replay.cpp

```diff
@@ -333,6 +364,12 @@ int main(int argc, char** argv)
   std::cout << "Using CUDA per-thread default stream.\n";
 #endif
 
+  auto const simulated_size =
+    static_cast<std::size_t>(args["size"].as<float>() * static_cast<float>(1u << 30u));
```

Comment:
Why convert to float and back?

Reply:
It's just a bit easier to specify it on the command line, e.g. `-s 13.5` instead of something like `-s 13824`.